app/serializers/search_serializer.rb

```diff
@@ -8,7 +8,8 @@ class SearchSerializer < ActiveModel::Serializer
   def serializable_hash
```

Comment:
I understand this is old code, and it definitely is not on you that it doesn't have documentation or unit test.  If possible, I'd like to be adding docs and tests, incrementally, as we touch old methods.